website/templates/project/project.mako

```diff
@@ -188,7 +188,7 @@
                 <span data-bind="css: icon"></span>
                 </p>
 
-                % if node['description'] or 'write' in user['permissions']:
+                % if node['description']:
```

Comment:
With this logic, contributors will not be able to change the description by clicking "No description".

If a project has no description, the desired behavior is:

- A contributor with "write" permission sees "Description: No description", which they can click on to set the description.
- Non-contributors do not see "Description". 

See if you can modify the logic to get this behavior, and make sure to verify the behavior with manual testing.